src/pages/Home/CustomizeBobaModalBody.tsx

```diff
@@ -3,18 +3,21 @@ import Form from 'react-bootstrap/Form';
 import Row from 'react-bootstrap/Row';
 import Col from 'react-bootstrap/Col';
 import { Container } from 'react-bootstrap';
-import { milkList, toppingsList } from 'assets/sampleBobaAPI';
 import RadioCheckboxGroup from 'components/RadioCheckboxGroup';
 import './CustomizeBobaModalBody.css';
 
 interface BobaModalProps {
+  milkList: { [key: string]: { name: string } };
+  toppingsList: { [key: string]: { name: string; price: number } };
```

Comment:
Remove this and move the props definiton into common for milk and toppings

Reply:
updated at 72649f73c89f41e75275f9309cd6fef3fd561f91